Shared/GeneralUtils.swift

```diff
@@ -33,6 +33,7 @@ open class PerformanceTimer {
         }
     }
 
+    @discardableResult
     public func stop() -> String? {
```

Comment:
If the only reason `@discardableResult` was added was for debug utility, then best to keep it and do `_ = X.stop()` at debug call point